src/screens/AddCash/AddCash.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+import { ScrollView, Keyboard } from 'react-native';
+import t from 'translations/translate';
+import styled from 'styled-components/native';
+import { useNavigation } from 'react-navigation-hooks';
+
+
+// utils
+import { fontSizes, appFont } from 'utils/variables';
+import { isValidFiatValue } from 'utils/validators';
+import { getCurrencySymbol } from 'utils/common';
+import { openInAppBrowser } from 'utils/inAppBrowser';
+import { rampWidgetUrl } from 'utils/fiatToCrypto';
+
+// compomnents
+import { Container } from 'components/modern/Layout';
+import { Footer } from 'components/Layout/Layout';
+import Button from 'components/Button';
+import TextInput from 'components/TextInput';
+import HeaderBlock from 'components/HeaderBlock';
+import Text from 'components/modern/Text';
+import Toast from 'components/Toast';
+
+// selectors
+import { useFiatCurrency } from 'selectors';
+
+
+const AddCash = () => {
+  const navigation = useNavigation();
+  const [value, setValue] = React.useState('0');
```

Comment:
Consider starting with empty input, as we already have placeholder with '$0' value.
```suggestion
  const [value, setValue] = React.useState('');
```